ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -1455,7 +1455,33 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
     }
   }
 
+  private cleanupParticipantData({data: ptData}: ParticipantData): void {
+    const SPECIFY = '_SPECIFY';
+    const OTHER_SPECIFY = '_OTHER_SPECIFY';
 
+    Object.keys(ptData).filter((key: string) => key.includes(SPECIFY)).forEach((key: string) => {
```

Comment:
This part of the code might look verbose, but it's totally built on top of the ParticipantData architecture. 
Please suggest better ways, I also don't like it :D, but anyway, at least it works (Up until now what I have checked).